<?xml version="1.0"?>
<launch>

    <node name="publish_eye_in_hand" pkg="calibration" type="publish_eye_in_hand"/>
    <node name="empty_service" pkg="smach_tutorials" type="empty_service_node.py" />

    <node name="task_manager" pkg="task_parser" type="task_parser_node" output="screen" />
    <node name="local_table_node" pkg="local_table" type="local_table_node" output="screen"/>
    
</launch>
